### ISSUE TITLE
test 2

### ISSUE THREAD

**Issue by grudra7714** [grudra7714/custom-slack-commands#3]

**Comment by grudra7714** [comment]
sa

**Comment by grudra7714** [comment]
saAS

**Comment by grudra7714** [comment]
@iamkrumbot

**Comment by grudra7714** [comment]
@iamkrumbot

**Comment by grudra7714** [comment]
comment

**Comment by grudra7714** [comment]
a

**Comment by grudra7714** [comment]
test

**Comment by grudra7714** [comment]
Krumbot should receive this and post to mobo channel

**Comment by grudra7714** [comment]
now this should appear as an attachment

**Comment by grudra7714** [comment]
now it would work

**Comment by grudra7714** [comment]
say what

**Comment by grudra7714** [comment]
how this looks ?

**Comment by grudra7714** [comment]
![man-161135_960_720](https://user-images.githubusercontent.com/12394597/38459851-37f286da-3a7d-11e8-856b-cd7fb91ee274.png)

**Comment by grudra7714** [comment]
attachment should be green

**Comment by grudra7714** [comment]
how about this

**Comment by grudra7714** [comment]
next issue would be big

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

Only be seen on desktop view (with map)
Not have selectors, icons, or other UI elements that would make this hangline movement acceptable.
That all being said (to log and track this that way there's some theory and knowlege behind it if someone questions the inconsistent spacing) it does look a bit quizzical with that much spacing, from the aesthetic standpoint. If there's no UX or functional conflicts with this suggested change (as listed above in my 1, 2) I'm okay with the movement of the text in the map popup header.

TL;DR

If we change that left-margin from the 27px to 8px (56px from the left edge of window) that should clean this spacing up AND shouldn't conflict with the 48px touch-area on the close icon in the map

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

Only be seen on desktop view (with map)
Not have selectors, icons, or other UI elements that would make this hangline movement acceptable.
That all being said (to log and track this that way there's some theory and knowlege behind it if someone questions the inconsistent spacing) it does look a bit quizzical with that much spacing, from the aesthetic standpoint. If there's no UX or functional conflicts with this suggested change (as listed above in my 1, 2) I'm okay with the movement of the text in the map popup header.

TL;DR

If we change that left-margin from the 27px to 8px (56px from the left edge of window) that should clean this spacing up AND shouldn't conflict with the 48px touch-area on the close icon in the map

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

Only be seen on desktop view (with map)
Not have selectors, icons, or other UI elements that would make this hangline movement acceptable.
That all being said (to log and track this that way there's some theory and knowlege behind it if someone questions the inconsistent spacing) it does look a bit quizzical with that much spacing, from the aesthetic standpoint. If there's no UX or functional conflicts with this suggested change (as listed above in my 1, 2) I'm okay with the movement of the text in the map popup header.

TL;DR

If we change that left-margin from the 27px to 8px (56px from the left edge of window) that should clean this spacing up AND shouldn't conflict with the 48px touch-area on the close icon in the map

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

**Comment by grudra7714** [comment]
Again, I'm hesitant and don't want to fork off, having some custom layout of these nav bars.

However, I understand that map popups are another element (compared to the app [framework] itself) with a different set of rules that more importantly would:

**Comment by grudra7714** [comment]
say what

**Comment by grudra7714** [comment]
sayyyy what

**Comment by grudra7714** [comment]
ds

**Comment by grudra7714** [comment]
dsa

**Comment by grudra7714** [comment]
d

**Comment by grudra7714** [comment]
dsa

**Comment by grudra7714** [comment]
dsa

**Comment by grudra7714** [comment]
s

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
text

**Comment by grudra7714** [comment]
What happens now

**Comment by grudra7714** [comment]
dsa

**Comment by grudra7714** [comment]
How this looks now ?

**Comment by grudra7714** [comment]
how about now

**Comment by grudra7714** [comment]
ohhhh
